src/views/Start.vue

```diff
@@ -125,39 +128,44 @@ export default {
         },
         {
           heading: "set your password",
-          text:
-            "You'll need this password to securely access your Umbrel from anywhere."
+          text: "You'll need this password to login to your Umbrel."
         },
         {
           heading: "confirm your password",
-          text:
-            "You'll need this password to securely access your Umbrel from anywhere."
+          text: "You'll need this password to login to your Umbrel."
         },
         {
           heading: "note down your secret words",
           text:
-            "On the next screen you will be shown 24 words. It's recommended that you write them down on a piece of paper and store it a safe place."
+            "On the next screen you will be shown 24 words. It's recommended that you write them down on a piece of paper and store it in a safe place."
         },
         {
           heading: "note down your secret words",
           text:
             'Remember, there is no "forget password" button. You will need these 24 words to recover your Umbrel node.'
         },
+        {
+          heading: "access from anywhere",
+          text:
+            "Even when you're not on your home network, you can access your Umbrel using Tor browser on the following URL."
+        },
         {
           heading: "one last thing",
           text:
-            "Since the Lightning Network and Umbrel are under heavy development, we recommend that you only store funds here that you're willing to lose. In other words, don't be too #reckless."
+            "Since the Lightning Network and Umbrel are under heavy development, it's recommended that you only store funds here that you're willing to lose. In other words, don't be too #reckless."
         },
         {
           heading: "🎉 that's it!",
           text:
-            "Congratulations! Your Umbrel is now running and synchronizing the Bitcoin blockchain."
+            "Congratulations! Your Umbrel is now set up and synchronizing the Bitcoin blockchain."
         }
       ],
       notedSeed: false,
       isRegistering: false,
       recover: false,
       recoverySeed: [],
+      onion:
+        "dj28fc0e8435fbae69764cdd83f412b6bb9b8451f6f2fa1c5a9d1084f372c5f1a6.onion",
```

Comment:
hardcoding it?

Reply:
This was just to quickly test the UI, it's still work in progress